app.py

```diff
@@ -36,22 +32,29 @@ def create_new_user():
     user = User(email=req['email'], password=hash)
     db.session.add(user)
     db.session.commit()
-#     not sure what to return here
-    return jsonify(req), 222
+    return jsonify({'user_id':user.id, 'email':user.email}), 222
 
 
 @app.route("/login", methods=['GET', 'POST'])
 def login_user():
         req=request.get_json()
         user=User.query.filter_by(email=req['email']).first()
-        print(user)
         if user and bcrypt.check_password_hash(req['password'], user.password):
                 profile=UserProfile.query.filter_by(user_id=user.id).first()
                 print(profile)
         # not sure what to return here either
         return jsonify('testing')
 
 
+@app.route("/create_user_profile", methods=['GET', 'POST'])
+def create_user_profile():
+        req=request.get_json()
+        user_profile=UserProfile(user_id=req['user_id'], name=req['name'], age=req['age'], proficiency=req['proficiency'])
```

Comment:
We could also use a session object to retrieve current user's id (future iteration)